docs/src/development/troubleshoot.md

```diff
@@ -49,12 +49,38 @@ The redeploy takes place after any scheduled activities (either *Running* or *Pe
 Despite the name, redeployment doesn't rerun the `deploy` hook, only the `post_deploy` hook.
 Both your `build` and `deploy` hooks are tied to individual commits in code.
 They're reused until another commit is pushed to the environment.
-See [more about hooks](../configuration/app/hooks.md) and their reuse.
+See [more about hooks](../configuration/app/hooks/_index.md) and their reuse.
 
-To rerun the `build` and `deploy` hooks, [manually trigger a build](../configuration/app/hooks.md#manually-trigger-builds).
+To rerun the `build` and `deploy` hooks, [manually trigger a build](#manually-trigger-builds).
 
 {{< /note >}}
 
+### Manually trigger builds
+
+To increase performance and keep applications the same across environments,
+Platform.sh reuses built applications if its code and build time configuration (variables and such) remain the same.
+
+There may be times where you want to force your application to be built again without changing its code,
+for example to test an issue in a build hook or when external dependencies change.
+To force a rebuild without changing the code,
+use an [environment variable](./variables/set-variables.md#create-environment-specific-variables).
+
+Assuming you want to do this for your `main` environment,
+first create a `REBUILD_DATE` environment variable:
+
+```bash
+platform variable:create -l environment -e main --prefix env: --name REBUILD_DATE --value "$(date)" --visible-build true
+```
+
+This triggers a build right away to propagate the variable.
+To force a rebuild at any time, update the variable with a new value:
+
+```bash
+platform variable:update -e main --value "$(date)" "env:REBUILD_DATE"
+```
+
+This forces your application to be built even if no code has changed.
```

Comment:
I didn't really look into that, but why would you want to delete the variable? We can do it in another issue if it makes sense.

Reply:
Because the "trick" works only once. Adding the variable only re-triggers the build once. Since the variable doesn't add much value to the project itself, it might be worth deleting after the trigger has been done.

Reply:
Oh, but you can update the date. That's what the last command does -- not create it, but update it. Every time you update it, it should retrigger a build. Because the value of the variable changes and it's available to the build, it should start the process. If it doesn't, that's a rather large bug.

Reply:
No you are right, I was wrong. Everything is working as expected.

Reply:
Glad it's working. If it's unclear, we can update it to be clearer.